# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/credentialmanagement/identity/moz.build
# Commit: 08eb9cf235bb
# Full Hash: 08eb9cf235bb5334ef01075d6148e1ea235adf56
# Author: Benjamin VanderSloot <bvandersloot@mozilla.com>
# Date: 2025-06-05 04:07:37
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D251617
# ==============================================================================

diff -r ff3135366145 -r 08eb9cf235bb dom/credentialmanagement/identity/moz.build
--- a/dom/credentialmanagement/identity/moz.build	Wed Jun 04 19:25:54 2025 +0000
+++ b/dom/credentialmanagement/identity/moz.build	Wed Jun 04 19:25:54 2025 +0000
@@ -13,16 +13,22 @@
     "IdentityCredentialSerializationHelpers.h",
     "IdentityNetworkHelpers.h",
     "NavigatorLogin.h",
+    "WebIdentityChild.h",
+    "WebIdentityHandler.h",
+    "WebIdentityParent.h",
 ]
 
 IPDL_SOURCES += [
-    "IPCIdentityCredential.ipdlh",
+    "PWebIdentity.ipdl",
 ]
 
 UNIFIED_SOURCES += [
     "IdentityCredential.cpp",
     "IdentityNetworkHelpers.cpp",
     "NavigatorLogin.cpp",
+    "WebIdentityChild.cpp",
+    "WebIdentityHandler.cpp",
+    "WebIdentityParent.cpp",
 ]
 
 include("/ipc/chromium/chromium-config.mozbuild")